library/Notifications/Common/Links.php

```diff
@@ -110,4 +110,12 @@ public static function moveRotation(): Url
     {
         return Url::fromPath('notifications/schedule/move-rotation');
     }
+
+    public static function ruleFilterSuggestionUrl(int $id): Url
+    {
+        return Url::fromPath(
+            "notifications/event-rule/complete",
+            ['_disableLayout' => true, 'showCompact' => true, 'id' => $id]
```

Comment:
Please do not add any model-related parameters here. This class should only provide basic links. I already had a discussion with Johannes about this topic. You can add more parameters while calling this function, or remove them altogether and create them in a specific class if required.